math.go

```diff
@@ -378,6 +379,78 @@ var (
 			}
 			return valueNode(nil, "avg", Null, nil), nil
 		},
+		"b64decode_no_padding": func(node *Node) (result *Node, err error) {
+			if node.IsString() {
+				if _, err := node.GetString(); err != nil {
+					return nil, err
+				} else {
+					sourceString, _ := node.GetString()
+					var result []byte
+					result, err = base64.StdEncoding.WithPadding(base64.NoPadding).DecodeString(sourceString)
+					if err != nil {
+						return nil, err
+					}
+					return valueNode(nil, "b64decode_no_padding", String, string(result)), nil
+				}
+			}
+			return valueNode(nil, "b64decode_no_padding", Null, nil), nil
+		},
+		"b64encode_no_padding": func(node *Node) (result *Node, err error) {
```

Comment:
Names for encoding functions should also be changed.
While in the `v0` there is no possibility to create a function with 2 or more arguments, we should create 2 functions for that. But better to add functions with more useful names, as a suggestion:

`b64encode` instead of `b64encode_std_padding`. This is the base behavior for `base64` application in *nix systems.
```
$ echo "foo" | base64
Zm9vCg==
```

`b64encoden` instead of `b64encode_no_padding`. This function (with `NO_PAD`) should not be used very often.

Reply:
Ok, I will rename the functions accordingly